src/coreclr/jit/gentree.cpp

```diff
@@ -18008,7 +18008,11 @@ bool GenTree::canBeContained() const
     }
     else if (OperIsHWIntrinsic() && !isContainableHWIntrinsic())
     {
+#ifdef TARGET_XARCH
         return isEvexEmbeddedMaskingCompatibleHWIntrinsic();
+#elif TARGET_ARM64
+        return HWIntrinsicInfo::IsEmbeddedMaskedOperation(AsHWIntrinsic()->GetHWIntrinsicId());
+#endif
```

Comment:
These seem like something we would want to just combine long term.

Can we just rename `isEvexEmbeddedMaskingCompatibleHWIntrinsic` to `isEmbeddedMaskingCompatibleHWIntrinsic` and have it do `return HWIntrinsicInfo::IsEmbeddedMaskedOperation(AsHWIntrinsic()->GetHWIntrinsicId());` on Arm64?